src/api/job.js

```diff
@@ -0,0 +1,56 @@
+const {Router} = require('express');
+const Job = require('../persistence/job');
+
+const router = new Router();
+
+// Admin Create a new job with post method
+router.post('/', async (request, response) => {
+  try {
+    const {
+      title,
+      salaryrange,
+      description,
+      tags,
+      company,
+      logoURL
+    } = request.body;
+    if (
+      !title ||
+      !salaryrange ||
+      !description ||
+      !tags ||
+      !company ||
+      !logoURL
+    ) {
+      return response
+        .status(400)
+        .json({message: 'All information must be provided!'});
+    }
+
+    const para = {
```

Comment:
Name variable clearly and abbreviation restrictions. We write the code for the team, not only for us. Let's think about how other members of our team read the code and understand it easily. For example, `parameters` in this case, not `para`, `pa`, or `p`.

Reply:
I understood, thank you.